Makefile

```diff
@@ -497,4 +498,10 @@ deploy-from-index-image:
 .PHONY: test-acceptance-with-bundle
 ## Run acceptance tests with the operator installed from a given index image and channel
 test-acceptance-with-bundle: deploy-from-index-image
-	$(Q)TEST_ACCEPTANCE_START_SBO=remote $(MAKE) test-acceptance
\ No newline at end of file
+	$(Q)TEST_ACCEPTANCE_START_SBO=remote $(MAKE) test-acceptance
+
+.PHONY: release-manifests
+## prepare file for Release
+release-manifests: setup-venv release-operator prepare-operatorhub-pr get-test-namespace
```

Comment:
is `get-test-namespace` really required for this target?

Reply:
For testing in the CI, it is required